content/install_maintain/installation/cfy-cluster-manager.md

```diff
@@ -173,7 +173,7 @@ cfy_cluster_manager remove [OPTIONS]
 
 
 ### Upgrading a {{< param product_name >}} cluster
-The {{< param product_name >}} cluster can be upgraded from v5.1.0 to v5.1.1 using the following command:
+The {{< param product_name >}} cluster can be upgraded from v5.1.0 to higher versions using the following command:
```

Comment:
@ofercloudify, I wrote "higher versions", but we can also specify the specific versions allowed for upgrading.